### DISCUSSION TITLE
hdr clipping

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14386] [Q&A] [answered]
Hi, I have a issue with hdr clipping. So when there is clipping and I lower the contrast inside mpv (`constrast=-24`), then the details are back. So they are hidden behind some threshold. Can I do something inside mpv to deal with this, like clamp the ouput or something like that?

Thank you :)

You can see the clipped clouds
![0 clipping](https://github.com/mpv-player/mpv/assets/139760266/16c01639-f145-413c-9790-360fa02e2083)

contrast set to -24 brings all the details back
![24 clipping](https://github.com/mpv-player/mpv/assets/139760266/c8f76993-30e0-4a3a-9c1e-c3d9df2877b0)

**Comment by kasper93** [comment]
```
vo=gpu-next
profile=high-quality
```

If it doesn't work, post sample, so we can test.

**Reply by geextahslex** [reply]
Here is the sample
[Aquaman HDR sample.zip](https://github.com/user-attachments/files/15945629/Aquaman.HDR.sample.zip)

**Reply by kasper93** [reply]
Cannot reproduce clipping with `vo=gpu-next`

**Reply by geextahslex** [reply]
What settings are you using for HDR tone mapping?

**Reply by kasper93** [reply, marked as answer]
I tested with `--no-config`, so default. Don't forget to adjust `--target-peak` for your display.
